includes/class-convertkit.php

```diff
@@ -244,7 +244,7 @@ public static function page_takeover() {
 
 			$landing_page_id = self::_get_meta( $queried_object->ID, 'landing_page' );
 
-			if ( '0' === $landing_page_id ) {
+			if ( '0' === $landing_page_id || empty( $landing_page_id ) ) {
```

Comment:
in what case would `$landing_page_id` be `"0"`?  Also, can we cover this change with a unit test?

Reply:
`$landing_page_id` is set to `'0'` when a WordPress post/page is saved _while the CK plugin is active_. If a post/page is saved while the plugin is not active, then when it is later activated, the `$landing_page_id` will be empty. @growdev's addition here checks for that so we don't show a PHP notice (which at least two customers have reported).